documentation/single-file-tools.md

```diff
@@ -0,0 +1,67 @@
+# Single File Diagnostic Tools
+
+All of our diagnostic tools have been distributed so far as [global tools](https://docs.microsoft.com/en-us/dotnet/core/tools/global-tools). These provide a great user experience as they provide clear ways to easily download, update, and configure the scope of the tools. However, these require a full .NET SDK which is often not available in some scenarios like production machines. For these scenarios we provide a single file distribution mechanism that only require a runtime to be available in the target machine. Other than the installation instructions, all functionality and usage remains the same. If you need help on how to use them, please refer to the [.NET Core diagnostic global tools](https://docs.microsoft.com/en-us/dotnet/core/diagnostics/#net-core-diagnostic-global-tools) documentation.
+
+## Requirements to Run the Tools
+
+- The tools distributed in a single file format are framework dependent applications. As such, they require a 3.1+ .NET Core runtime to be available. This means one of the following:
+  - The runtime was installed globally in a well-known location using one of the different installer technologies documented in the [official .NET download page](https://dotnet.microsoft.com/download).
+  - Downloading the binary archives from the [official .NET download page](https://dotnet.microsoft.com/download) and extracting them to a directory. Afterwards, please set `DOTNET_ROOT` to the path of the extraction.
+  - Using the [dotnet-install scripts](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script) and setting `DOTNET_ROOT` to the installation directory. This mechanism is not recommended for development or production environments, but it's ideal for environments like CI machines.
+  - The workload is being run in a container with the runtime available. This could be either because you are running a workload on our [official .NET runtime containers](https://hub.docker.com/_/microsoft-dotnet-runtime) or you built a custom image that installs it at build time.
+- These tools are bundles that will inflate on the target machine and will extract files to disk as necessary. The extraction folder is by default within the `TEMP` directory. In case `TEMP` is not available in the target environment or you want to control such directory for reasons such as clean up, please set `DOTNET_BUNDLE_EXTRACT_BASE_DIR` to a path that can be used in lieu of `TEMP`.
```

Comment:
```suggestion
- These tools are bundles that will inflate on the target machine and will extract files to disk as necessary. The extraction folder is by default within the `TEMP` directory. In case `TEMP` is not available in the target environment or there's a desire to control such directory for reasons such as clean up, set `DOTNET_BUNDLE_EXTRACT_BASE_DIR` to a path that can be used in lieu of `TEMP`.
```